zingolib/src/utils/conversion.rs

```diff
@@ -29,14 +29,13 @@ pub fn txid_from_hex_encoded_str(txid: &str) -> Result<TxId, TxIdFromHexEncodedS
     Ok(TxId::from_bytes(txid_bytes))
 }
 
-pub(crate) fn address_from_str(
-    address: &str,
-    chain: &ChainType,
-) -> Result<Address, ConversionError> {
+/// Convert a &str to an Adddress
+pub fn address_from_str(address: &str, chain: &ChainType) -> Result<Address, ConversionError> {
     Address::decode(chain, address)
         .ok_or_else(|| ConversionError::InvalidAddress(address.to_string()))
 }
 
-pub(crate) fn zatoshis_from_u64(amount: u64) -> Result<NonNegativeAmount, ConversionError> {
+/// Convery a valid u64 into Zatoshis.
```

Comment:
I like it. typo though .. lets see if i can spell `convert` properly. You passed the test and corrected my correction on `address_from_str`